rest/src/main/java/io/syndesis/rest/v1/operations/Validating.java

```diff
@@ -0,0 +1,63 @@
+/**
+ * Copyright (C) 2016 Red Hat, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.syndesis.rest.v1.operations;
+
+import java.util.Set;
+import java.util.stream.Collectors;
+
+import javax.validation.ConstraintViolation;
+import javax.validation.Validator;
+import javax.validation.constraints.NotNull;
+import javax.ws.rs.Consumes;
+import javax.ws.rs.POST;
+import javax.ws.rs.Path;
+import javax.ws.rs.Produces;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+import javax.ws.rs.core.Response.Status;
+
+import io.swagger.annotations.ApiResponse;
+import io.swagger.annotations.ApiResponses;
+import io.syndesis.model.WithId;
+import io.syndesis.model.validation.AllValidations;
+
+public interface Validating<T extends WithId<T>> extends Resource {
+
+    Validator getValidator();
+
+    @POST
+    @Path(value = "/validation")
+    @Consumes("application/json")
+    @Produces(MediaType.APPLICATION_JSON)
+    @ApiResponses({//
+        @ApiResponse(code = 200, message = "All validations pass"), //
+        @ApiResponse(code = 409, message = "Found violations in validation", responseContainer = "Set",
```

Comment:
Now I think about it, I think that 400 would be a better response as this is potentially not only testing for uniqueness (where 409 would make perfect sense).